### DISCUSSION TITLE
Deleting vertexes from line on touch device.

### DISCUSSION THREAD

**Discussion by panaaj** [openlayers/openlayers#16137] [Q&A]
When using the modify interaction to modify a line, using the ctrl modifier key allows vertexes to be deleted. How is this achieved on a touch device?

**Comment by ahocevar** [comment]
There is a [removePoint()](https://openlayers.org/en/latest/apidoc/module-ol_interaction_Modify-Modify.html#removePoint) method on the `Modify` interaction. Just call that when the user taps the point that shall be deleted.
